opensrp-chw/src/kk/java/org/smartregister/chw/interactor/AncHomeVisitInteractorFlv.java

```diff
@@ -969,4 +982,66 @@ public void onPayloadReceived(BaseAncHomeVisitAction baseAncHomeVisitAction) {
             Timber.v("onPayloadReceived");
         }
     }
+
+    private class ChwObservationsAction implements BaseAncHomeVisitAction.AncHomeVisitActionHelper {
+        private String anyone_presence;
```

Comment:
This needs initialization 
```suggestion
        private String anyone_presence = "";
```